### ISSUE TITLE
Can we add text in the spinner

### ISSUE THREAD

**Issue by nmulakaluri** [urish/angular-spinner#57]
We added us spinner in our project and a preloader comes whenever there is a service call. We managed to fade the page while preloading. now there is a requirement that we need add a text under the loading spinner. is it possible? I think it is a cool feature to have if you can extend this directive that would be great.

**Comment by Attrash-Islam** [comment]
Hi @nmulakaluri ,
Is this still relevant?

Thanks

**Comment by MaxDhn** [comment]
Hi  @Attrash-Islam ,
I'm also interested by this feature :+1:

**Comment by Attrash-Islam** [comment]
Okay MaxDhn,
I'll support that as soon as possible. Thoughts in this thread is also helpful. Starting from you Max

**Comment by nmulakaluri** [comment]
@Attrash-Islam Yes this is still a valid scenario. Please let me know if you have a solution to this

**Comment by Attrash-Islam** [comment]
I'm sorry pushing this issue in a delay because of work pressure this period. But I'll try my best as soon as possible
